docs/controls/geoman_draw_control.rst

```diff
@@ -1,10 +1,11 @@
 Geoman Draw Control
-============
+===================
 
-``GeomanDrawControl`` allows one to draw various shapes on the map. 
+``GeomanDrawControl`` allows one to draw various shapes on the map.
```

Comment:
These two lines in seem unrelated to the commit they are in.

Reply:
You are right, these two minor changes came when I rendered the fix to the list locally, since my IDE drops trailing spaces and there was a warning in Sphinx about the nr of `=` that I took the chance of fixing.

I should probably have at least added an additional line in the commit message stating stg like "Also fix some minor cosmetic RST formatting."

If you prefer to have a more accurate commit history, I am happy to force-push them as individual changes. Let me know, no big deal for me.